config.js

```diff
@@ -127,6 +127,12 @@ const queueStatsActivityLogsInterval = 60000;
 // sent to collector.
 const encryptionAlgoForCollector = 'aes-256-cbc';
 
+// strings to represent resource type for caching. used by getHash() function
+// to generate keys for redis
+const resourceTypes = {
```

Comment:
Is there maybe another place to get that string from? (I'm not sure there is a reasonable place, but it seems like there would already be. If you haven't checked, please do, otherwise I am ready to approve this.)

Reply:
done!